src/scrabble.js

```diff
@@ -1,5 +1,127 @@
-function scrabble() {
+function scrabble(string) {
+  // let score = 0
 
+  if (!string) {
```

Comment:
Didn't know this was a way to check if it's a string! Great job!

Reply:
Also didn't know - very clean, big fan!